### PR TITLE
Add release documentation

### PR DESCRIPTION
**Pull request by m273d15**
We don't perform the old release process (with multiple roles etc.) anymore.
Therefore I removed the old documentation and created a description of the steps which are required for the release process.

### CODE REVIEW

File: docs/contribute/processes/release.md [hunk 1]
```diff
@@ -2,681 +2,163 @@
...
+**Artifacts:**
+* Update Site
+* Drop-in
+
+#### Update Site
+
+* Open an eclipse instance which contains a working [Saros development setup](https://www.saros-project.org/contribute/development-environment.html)
+* Open the project `Saros Update Site`
+* Open the file `site.xml`
+* Make sure that the category `DPP` contains the current feature `saros.feature*`
```
Comment (tobous) [inline docs/contribute/processes/release.md]: Is this really still `DPP`? How much work is it to change this?

File: docs/contribute/processes/release.md [hunk 2]
```diff
@@ -2,681 +2,163 @@
...
+* Create a pull request that contains the new update site
+* Merge the pull request
+* Check that the update site is deployed successfully
+
+#### Eclipse Marketplace
+
+**What is released via this channel?**
+The Saros for Eclipse plug-in is released via this channel, but you don't have to release the artifact explicit via
+this channel, because the Marketplace uses the **Update Site**.
+As long as the infrastructure does not change (as the update site location) just update **Version number** information
```
Comment (tobous) [inline docs/contribute/processes/release.md]:
```suggestion
As long as the infrastructure does not change (like the update site location) just update **Version number** information
```

File: docs/contribute/processes/release.md [hunk 3]
```diff
@@ -2,681 +2,163 @@
...
+* Open bash
+* Navigate to the saros repository dir
+* Execute the command `./gradlew sarosIntellij`
+* You find the zip here: `./build/distribution/intellij/saros.inteliij.zip`
+
+## How do I test the artifacts?
+
+As mentioned before you should already executed all unit and stf tests (which has to be successful).
+
+The release artifacts are manually tested with at least one other persion:
```
Comment (tobous) [inline docs/contribute/processes/release.md]:
```suggestion
The release artifacts are manually tested with at least one other person:
```

File: docs/contribute/processes/release.md [hunk 4]
```diff
@@ -2,681 +2,163 @@
...
+in order to change the update site.
+
+**What is released via this channel?**
+We release the update site (which contains the Eclipse plug-in jars and additional resources) via this channel.
+
+**Login process**
+
+* Login with the `saros-infrastructure` user
+
+** Release process**
```
Comment (tobous) [inline docs/contribute/processes/release.md]: Additional space breaks formatting.

```suggestion
**Release process**
```

File: docs/contribute/processes/release.md (hunk 1 above)
Comment (m273d15) [reply docs/contribute/processes/release.md]: Changing the category should be simply, but we have to check whether the update still works as expected (I assume that it works, but we should test it). The question is whether we change the name or remove the category (a category contains multiple features, but we have just one feature (saros.eclipse)). If you don't use a category, the update-site (during the installation process) shows the category `Unknown` (or something similar). I open a separate issue in order to track this.

Comment (m273d15) [reply docs/contribute/processes/release.md]: See #557

**Pull request merged**